charts/k8s-monitoring/tests/destination_validations_test.yaml

```diff
@@ -76,6 +76,27 @@ tests:
     asserts:
       - notFailedTemplate: {}
 
+  - it: detects if you're using an otlp destination with an invalid protocol
+    set:
+      cluster:
+        name: test-cluster
+      destinations:
+        - name: my-destination
+          type: otlp
+          url: https://otlp-gateway-test.grafana.net
+          protocl: magic
+    asserts:
+      - failedTemplate:
+          errorMessage: |-
+            execution error at (k8s-monitoring/templates/validations.yaml:5:4): 
+            Destination #0 (my-destination) is using Grafana Cloud OTLP gateway but has incorrect protocol 'grpc (default)'. The gateway requires 'http'.
```

Comment:
should this show the user defined protocol in the error message?  `but has incorrect protocol 'magic'`

Reply:
🤦🏻

Reply:
fixed